modules/common/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/common/config/Config.scala

```diff
@@ -85,7 +85,7 @@ object Config {
     def output: Shredder.Output
   }
   object Shredder {
-    final case class Batch(input: URI, output: Output) extends Shredder
+    final case class Batch(input: URI, output: Output, snsTopic: Option[String]) extends Shredder
```

Comment:
Sounds like `snsTopic` should be part of `Output`

Reply:
I've addressed your config related changes during [config refactoring](https://github.com/snowplow/snowplow-rdb-loader/pull/609).